### DISCUSSION TITLE
Is there a new order listing event in seaport？

### DISCUSSION THREAD

**Discussion by kkthmyh** [ProjectOpenSea/seaport#651] [General]
I want to listening a new order listing event，but I can not find this event in seaport，someone can tell me how to do？

**Comment by ryanio** [comment]
There is an on-chain `validate` event which you can deconstruct the tx calldata to get the order details.

To get all orders (on and off chain) you can use the OpenSea API: https://docs.opensea.io/v2.0/reference/retrieve-listings
